README.rst

```diff
@@ -14,18 +14,78 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
```

Comment:
What do you mean by independent? How do you explain https://github.com/ome/omero-figure/blob/master/omero_figure/views.py#L26-L35 and https://github.com/ome/omero-figure/blob/master/omero_figure/urls.py#L20 ?